### PR TITLE
net-im/skypeforlinux: version bump to 8.14.0.10

### PR DESCRIPTION
**Pull request by karolgrudzinski**
Bug: https://bugs.gentoo.org/646088
Package-Manager: Portage-2.3.20, Repoman-2.3.6

### CODE REVIEW

**Comment by EvaSDK** [comment]
Please use xdg.eclass directly if there's nothing there you need to avoid specifically. xdg-utils.eclass is only for corner cases.

**Comment by mgorny** [comment]
@evasdk, please don't give advices contrary to what have been suggested before, especially when there's no technical reason to proliferate horribly convoluted base.eclass clones.

File: net-im/skypeforlinux/skypeforlinux-8.14.0.10.ebuild [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+	for res in 16 32 256 512; do
+		newicon -s ${res} usr/share/icons/hicolor/${res}x${res}/apps/skypeforlinux.png skypeforlinux.png
+	done
+
+	domenu usr/share/applications/skypeforlinux.desktop
+
+	if use pax_kernel; then
+		pax-mark -m "${ED%/}"/opt/skypeforlinux/skypeforlinux
+		pax-mark -m "${ED%/}"/opt/skypeforlinux/resources/app.asar.unpacked/node_modules/slimcore/bin/slimcore.node
+		eqawarn "You have set USE=pax_kernel meaning that you intend to run"
```
Comment (mgorny) [inline net-im/skypeforlinux/skypeforlinux-8.14.0.10.ebuild]: eqawarn is meant for developer-oriented warnings, i.e. when eclasses say that ebuild is broken. Here you want `ewarn`.

**Comment by EvaSDK** [comment]
@mgorny, please don't compare xdg.eclass to base.eclass. Their scope isn't anything comparable and I don't see how convoluted it is (ftr: https://gitweb.gentoo.org/repo/gentoo.git/tree/eclass/xdg.eclass). If devs want to avoid dependencies of the eclass for special cases this is fine but the risk of missing cache updates should be considered, especially for desktop apps.
Also please add a reference to that advice you are talking about as I have missed it.

**Pull request closed without merging**